scripts/us_bea/states_gdp/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "USStatesQuarterlyGDP",
+            "curator_emails": [
+                "fpernice@google.com"
+            ],
+            "provenance_url": "https://www.bea.gov/data/gdp/gdp-state",
+            "provenance_description": "Quarterly data on inflation-adjusted GDP at a state level in the United States from the US Bureau of Economic Analysis.",
+            "scripts": [
+                "import_data_test.py",
+                "validate_import.py",
+                "import_data.py",
+                "import_industry_data_and_gen_mcf.py"
+            ],
+            "import_inputs": [
+                {
+                    "template_mcf": "states_gdp.tmcf",
+                    "cleaned_csv": "states_gdp.csv",
+                    "node_mcf": "states_gdp_statvars.mcf"
+                },
+                {
+                    "template_mcf": "states_industry_gdp.tmcf",
+                    "cleaned_csv": "states_industry_gdp.csv",
+                    "node_mcf": "states_gdp_industry_statvars.mcf"
+                }
+            ],
+            "cron_schedule": "0 10 30 1,4,7,10,12 *"
```

Comment:
Typically we choose a random minute so that the system is not overloaded at the beginning of the hour.

Some systems permit specifying "?" for a system chosen random value, but I do not think Google Cloud Scheduler (?Cron?) supports this.

Reply:
[https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules](https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules) links to [https://man7.org/linux/man-pages/man5/crontab.5.html](https://man7.org/linux/man-pages/man5/crontab.5.html), which does not seem to support `?`. We could choose a random minute manually.

Reply:
changed to 10:30